### ISSUE TITLE
Where is the training set?

### ISSUE THREAD

**Issue by Dobatymo** [MarkWuNLP/MultiTurnResponseSelection#18]
> We release Douban Conversation Corpus, comprising a training data set, a development set and a test set

I can only see the test set (test.txt) and a small sample of the training set. Where is the rest of the training (and validation) set?

**Comment by MarkWuNLP** [comment]
Hi, please email me (wumark@126.com), and I will send you the rest data

**Comment by yangsheng-12051** [comment]
你好，项目主页里Resource file: https://1drv.ms/u/s!AtcxwlQuQjw1jGn5kPzsH03lnG6U、Worddict file: https://1drv.ms/u/s!AtcxwlQuQjw1jGrCjg8liK1wE-N9和Word embedding: https://1drv.ms/u/s!AtcxwlQuQjw1jF0bjeaKHEUNwitA都下载不了，能麻烦发我邮箱吗，谢谢。
邮箱地址：ysheng12051@163.com
